ast/expressions.go

```diff
@@ -137,7 +138,13 @@ func (n *BinaryOperationExpr) Format(w io.Writer) {
 	fmt.Fprint(w, " ")
 	n.Op.Format(w)
```

Comment:
WriteString will not consider the % escape, it must be something in fmt.Sprint that %% => %